doc/cc65.sgml

```diff
@@ -1388,7 +1388,10 @@ parameter with the <tt/#pragma/.
   the <tt/Y/ register if it wraps any variadic functions (they have "<tt/.../"
   in their prototypes).
 
-  The identifier is an 8-bit number that's set into <tt/tmp4/.
+  The identifier is an 8-bit number that's set into <tt/tmp4/. If the identifier
+  is "bank", then a <tt><htmlurl url="ca65.html#.BANK" name=".bank"></tt> operator will be used
+  to determine the number from the bank attribute defined in the linker config,
+  see <htmlurl url="ld65.html#MEMORY" name="Other MEMORY area attributes">.
```

Comment:
please let me know if there is a better way to reference across documents - htmlurl is a bit kludgy

Reply:
I think it needs to be briefly mentioned that this implies ignoring the top 24 bits of the 32-bit bank attribute.